### DISCUSSION TITLE
Gogs 0.12.4 has been released!

### DISCUSSION THREAD

**Discussion by unknwon** [gogs/gogs#6736] [Announcements]
Hello everyone,

Hope you have pleasant experiences with Gogs so far, and we’re excited to announce that Gogs 0.12.4 is released!

You can find binary downloads on our [website](https://gogs.io/docs/installation/install_from_binary) and the [GitHub release](https://github.com/gogs/gogs/releases/tag/v0.12.4) page.

The CHANGELOG for this release is also available on the [GitHub release](https://github.com/gogs/gogs/releases/tag/v0.12.4) page.

Please enjoy and happy coding!

Please comment on this thread if you spot any packaging issues for the release. For general feature requests and bug reports, please use our [curated issue forms](https://github.com/gogs/gogs/issues/new/choose) :)

### Backwards compatibility

You may upgrade your installation directly to this version if you're running Gogs 0.8.5 or newer.

**Comment by WEiQuNBrian** [comment]
Hi Joe,

I tried to upgrade to 0.12.4 onmy centOS7.9 server but failed due to an error of "GLIBC2 .18 not found".

Do I have to upgrade to GLIBC2.18 in order to use the latest version of Gogs?

Thanks

**Reply by unknwon** [reply]
Hey @WEiQuNBrian, the Linux AMD64 binary was compelled on Ubuntu 20.04 so it makes sense that it uses higher version of GLIBC, I am just not sure if it's worth you to upgrade, or should I continue fall back to use 16.04 (which is pretty pretty old...) 🤔

**Reply by unknwon** [reply]
Please follow along on the https://github.com/gogs/gogs/issues/6752 for future updates.

**Comment by tlamer** [comment]
Hello @unknwon!

Thanks for the release! I've noticed that the gogs/gogs-rpi was not updated to the newly released version yet. Is there any plan to provide the updated image for arm as well?

Thanks & kind regards!

**Reply by unknwon** [reply]
Hey @tlamer! All of `gogs/gogs:latest`, `gogs/gogs:0.12.4` and `gogs/gogs:0.12` are [multi-arch images](https://docs.docker.com/desktop/multi-arch/), which means if you just pull these images, `linux/arm/v7` will be automatically downloaded for you on a Raspberry Pi. LMK if you have other questions!

**Reply by tlamer** [reply]
That's awesome! Thanks for the information. I wasn't aware of the multi-arch images!

**Reply by unknwon** [reply]
No problem! This is also a new thing for me 😀

**Comment by DeeJay** [comment]
Please see #6750  - I didn't see your request to comment here before creating a new Issue.

**Reply by unknwon** [reply]
Thanks @DeeJay, I'm taking a look!
